src/nl/surf/eduhub/validator/service/authentication.clj

```diff
@@ -116,3 +116,10 @@
       (if-let [token (bearer-token request)]
         (handle-request-with-token request f (authenticator token))
         (f request)))))
+
+(defn wrap-allowed-clients-checker [f allowed-client-id-set]
+  {:pre [(set? allowed-client-id-set)]}
+  (fn [{:keys [client-id] :as request}]
+    (if (and client-id (not (allowed-client-id-set client-id)))
```

Comment:
We willen geen toegang voor not-authenticted parties dus iets als

```
 (if (allowed-client-id-set client-id)
    (f request)
     {:body "Unknown client id" :status http-status/forbidden}
```

Reply:
Ok, aangepast